libraries/AP_HAL_ChibiOS/hwdef/DroneerF405/hwdef-bl.dat

```diff
@@ -0,0 +1,39 @@
+# hw definition file for Droneer
+# tested on the Droneer F405
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1054
```

Comment:
This ID is used. Please reserve an unused id

Reply:
@DroneerUAV to be clear, you need to make an entry in `Tools/AP_Bootloader/board_types.txt`, reserving an ID for your board.  You then you use either that numeric ID in here, or the board name you've used in `board_types.txt`